lib/python/rose/stem.py

```diff
@@ -359,11 +360,24 @@ def _check_suite_version(self, fname):
         if not suite_rose_stem_version == ROSE_STEM_VERSION:
             raise RoseStemVersionException(suite_rose_stem_version)
 
+    def _get_localhost(self):
+        """Return the local host on which the rose-stem command was invoked."""
+        return socket.gethostname()
+
+    def _prepend_localhost(self, url):
+        """Prepend the local hostname to urls which do not point to repository
+        locations."""
+        if ('svn:' not in url and 'fcm:' not in url and 'http:' not in url
+                and 'https:' not in url and 'svn+ssh:' not in url):
```

Comment:
I'm not sure that logic is quite correct, I think it's `if ':' not in url...` at which point it's probably unnecessary as if there's no `:` in `url` the split won't contain one of those four strings?